client/rpc-core/src/types/transaction_request.rs

```diff
@@ -119,3 +120,101 @@ impl From<TransactionRequest> for Option<TransactionMessage> {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
+
```

Comment:
```suggestion
```

Reply:
where are this style/convention guidelines defined or described and, if they are enforced, why are not part of the linting?

Reply:
it's just my taste, there is no relevant rustfmt or clippy config in my impression

Reply:
If it's just your preference, it's a cosmetic one, and there is no improvement at all in any area, there is no change to be done.